app/javascript/src/components/Dashboard/Notes/utils.js

```diff
@@ -0,0 +1,10 @@
+import dayjs from "dayjs";
+
+var relativeTime = require("dayjs/plugin/relativeTime");
+
+dayjs.extend(relativeTime);
```

Comment:
Go through https://academy.bigbinary.com/learn-react/pages/miscellaneous-tips#:~:text=How%20to%20extend%20day.js%3F very well and make necessary changes.

The point of writing and documenting standards in the React Best Practices book is to ensure that we avoid redundant review comments like the one I am giving right now.

If you don't agree with what's mentioned in the book, then it's totally fine and it's acceptable to keep this change. But you ought to initiate a discussion and make your strong points against the content from the book. If there's a need to update the content in the book then we can do that.

If you feel the content in the book makes sense and if you are not following those standards in your code, then that's not acceptable. I understand that folks miss things out. Thus the self-review process. Read: https://handbook.neetokb.com/articles/git-and-github-essentials#self-review-process

Reply:
Sorry for the mistake. I have made the necessary changes and will take care to avoid the same mistakes in future.

Reply:
@KrishnapriyaSkk https://www.loom.com/share/0a62ad2ca68d48969ae6994b6ae51824

Reply:
Hi. I have recorded a loom regarding the same here. 
https://www.loom.com/share/320cfd5dbd034d5b9376e676c5616140

Reply:
I am posting this loom here since it is related to this conversation. 
https://www.loom.com/share/60bb89420d224aeda35ab74351f3afce

Reply:
@KrishnapriyaSkk I see that you are using `docket-compose` for running the servers. Can you try it the barebones way by running the rails server in a separate terminal and webpack-dev-server in a separate server and see if that fixes the issue for you?

Essentially talking about restarting the webpack server.

Reply:
@KrishnapriyaSkk Can you put up a video showing the full stack trace and what all you have done in order to rectify the issue? That will provide me more context to debug this issue. Thanks.

Reply:
@yedhink Hi here is a video showing the issue.
https://www.loom.com/share/1cede332a99844fa9d2ea42f05c188f7

Reply:
@KrishnapriyaSkk Is your VSCode integrated terminal Rosetta enabled terminal?

Reply:
Few steps I can suggest. Ensure you are running the following from a separate(not in VSCode) terminal instance, that is Rosetta enabled, from the root of the project:
- Remove `postgresql` from your system and `libpq`. Uninstall the whole thing via brew
- Reinstall both of them.
- Then after that start the postgres services via brew and check the status of the services and ensure it's working.
- Then remove all `pg` gems. `gem uninstall pg` and remove all versions
- Then from the root of your project, in the Rosetta terminal, run `bundle install` and lemme know what happens.

Reply:
Okay let me try these steps. Thank you.

Reply:
I had disabled rosetta because it was throwing an error. Now I have enabled again, but brew installation fails with following error:

brew install postgresql
Running `brew update --auto-update`...
Warning: Use postgresql@14 instead of deprecated postgresql
Warning: Use postgresql@14 instead of deprecated postgresql
Error: Cannot install under Rosetta 2 in ARM default prefix (/opt/homebrew)!
To rerun under ARM use:
    arch -arm64 brew install ...
To install under x86_64, install Homebrew into /usr/local.

Reply:
I'm able to run the server now. `bundle install `worked fine. But getting this error on running the server.
<img width="1440" alt="Screenshot 2022-09-22 at 9 07 43 PM" src="https://user-images.githubusercontent.com/44946937/191809396-9666203c-c937-4731-acc4-9912180a9eee.png">

Reply:
The server is running now. I was able to do `bundle install`. But now the page loads an error:
<img width="1440" alt="Screenshot 2022-09-22 at 9 14 34 PM" src="https://user-images.githubusercontent.com/44946937/191810565-11a86a3b-b629-4d6d-9844-c1aa587e6f19.png">

Reply:
Hi. I am facing a slightly different issue now. I have recorded a video tracking the files as you mentioned. 

https://www.loom.com/share/23028af85aed437a817b883c49ef3abb

Reply:
@KrishnapriyaSkk Raise a separate PR from a separate branch, in which you have the exact same code where the error shown in your video can be reproduced.

To ensure that it's the exact same code, once you make the PR, first clone the repo into say `/tmp/neetoui` then checkout the branch where issue exists and run the server and check whether same issue persists.

The reason for even going to this extent of a verification is because folks have been making PRs and assigning it for me to debug but I can't reproduce any issue because they haven't pushed the code where issue actually persists.

Once everything is verified, assign me that new PR. I will take a look into the issue.

Reply:
Hi. The earlier issue with docker seem to be gone now for me and I am able to run the server. Is it okay if i work with it for now and once I commit fixes for all other issues, I take up this error again and handle it?

Reply:
Hi. I tried cloning the repo again till the last merge. Now it seems to be working fine. Thank you so much for the help.

Reply:
> @yedhink I am posting this loom here since it is related to this conversation. Please do help me out, incase I am missing something . https://www.loom.com/share/60bb89420d224aeda35ab74351f3afce

But now I am facing the same issue.